sources/quest_2/chapter_3/step_14/step_14_start.move

```diff
@@ -0,0 +1,60 @@
+use std::signer;
+
+module robinson::my_shore {
+
+    
+    struct GlobalData has key {
+        nb_tree: u8,
+        has_river: bool,
+        shore_location: address,
+        animal: vector<u8>,
```

Comment:
Here you should had the instructions too like : 

// Add a island_name var with type String 

Same for the String lib imports, we should put instructions where code should be added with tips on name and args to help the code check of the app !

Reply:
Ok! Noted.